.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+# .github/workflows/ci.yml
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    uses: catalyst/catalyst-moodle-workflows/.github/workflows/ci.yml@main
+    #secrets:
+      # Required if you plan to publish (uncomment the below)
+      # moodle_org_token: ${{ secrets.MOODLE_ORG_TOKEN }}
+    with:
+      disable_behat: true
+      extra_plugin_runners: 'moodle-plugin-ci add-plugin catalyst/moodle-tool_crawler --branch MOODLE_310_STABLE'
```

CI is failing. Hmm just a hunch, but potentially the branch flag should be before the repo reference
e.g.
```suggestion
      extra_plugin_runners: 'moodle-plugin-ci add-plugin --branch MOODLE_310_STABLE catalyst/moodle-tool_crawler'
```